spec/spec_helper.rb

```diff
@@ -17,4 +19,16 @@
   config.infer_base_class_for_anonymous_controllers = false
   config.order = "random"
   config.include FactoryGirl::Syntax::Methods
+
+###################### DatabaseCleaner Setting ######################
```

Comment:
Style/CommentIndentation: Incorrect indentation detected (column 0 instead of 2).